### DISCUSSION TITLE
Setup permissions - screen freeze

### DISCUSSION THREAD

**Discussion by AntOonin** [eduardofilo/RG350_adam_image#243] [Hardware problems?]
Dear all,

I have a repeated issue with the Adam image, the boot seems to be closed to being finalised. The grey bar is full, but a final message appears "setup permissions...". The problem is that the screen gets stuck on this message and nothing happens.
I first tried version 1.4.2 of the image, as it didn't work I then tried version 1.41. I let the boot run all night on 1.4.1 thinking that maybe it could take some time but this morning I still had the same message...

I didn't find an equivalent problem in the discussions and various FAQs. If you have a tip I would be interested!

Thanks a lot and thanks for all the work done

**Comment by neilswann80** [comment]
A few Qs...
- What device do you have?
- What method/software did you use to flash?
- Have you tried a different microsd card?

**Reply by AntOonin** [reply]
Well I tried your third option and, actually, it works perfectly and really smoothly. Now I feel a bit stupid for not having tried this before... ^^

I guess now I just have to enjoy it all !

Thanks a lot for your support and reactivity. Have a nice day

**Reply by neilswann80** [reply]
Glad to hear it... happy gaming!
